recipe/meta.yaml

```diff
@@ -46,8 +49,10 @@ test:
   # The only reason not to run the tests on Windows is that
   # they terminate with an interactive prompt. See also
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
+  # TestSocket.test_large_send is skipped on upstream CI for pypy
   commands:   # [not win]
-    - py.test --pyargs zmq.tests.test_socket  # [not win]
+    - py.test --pyargs zmq.tests.test_socket  # [not win and python_impl == 'cpython']
+    - py.test  -k 'not TestSocket.test_large_send' --pyargs zmq.tests.test_socket  # [not win and python_impl == 'pypy']
```

Comment:
@minrk, I tried skipping here. Not sure what the correct syntax is

Reply:
using just the function name works. Let's see if it passes this time.